src/docfx/build/Builder.cs

```diff
@@ -59,7 +59,7 @@ public void Build(params string[] files)
 
         private DocsetBuilder[] LoadDocsets()
         {
-            var docsets = ConfigLoader.FindDocsets(_errors, _workingDirectory, _docsetPackage, _options);
+            var docsets = ConfigLoader.FindDocsets(_errors, _docsetPackage, _options);
```

Comment:
The package in `Builder` is not specific to a docset, consider rename it to `_package`.

```suggestion
            var docsets = ConfigLoader.FindDocsets(_errors, _package, _options);
```

Reply:
would `package` be too general for in this context?

---
In reply to: [535802282](https://github.com/dotnet/docfx/pull/6840#discussion_r535802282) [](ancestors = 535802282)